client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionDialog.tsx

```diff
@@ -157,16 +180,27 @@ const InteractionDialog = (props: Props) => {
                     });
                 }
             }
-            if (Boolean(interactionStartTime && interactionEndTime) || isUnknownTime) {
-                const data = methods.getValues();
+
+            if (isRepetitiveInvalid) {
+                methods.setError(InteractionEventDialogFields.IS_REPETITIVE, {
+                    type: 'manual',
+                    message: isRepetitiveFieldMissing
+                        ? repetitiveFieldMissingMessage
+                        : repetitiveWithoutDatesSelectedErrorMessage
+                });
+            }
+
+            const canSubmit = (!!(interactionStartTime && interactionEndTime) || isUnknownTime) && !isRepetitiveInvalid;
+            if (canSubmit) {
                 delete data.privateHouseAddress;
                 onSubmit(data);
             }
         })
 
     return (
         <FormProvider {...methods}>
-            <Dialog classes={{ paper: classes.dialogPaper }} open={isOpen} maxWidth={false}>
+            <DevTool control={methods.control}/>
```

Comment:
is it supposed to stay? or is it just for debugging?

Reply:
its only visible in dev so thought it would be useful to keep it... could remove it if you prefer